superset-frontend/webpack.config.js

```diff
@@ -341,6 +302,8 @@ if (isDevMode) {
     },
     historyApiFallback: true,
     hot: true,
+    injectClient: false,
+    injectHot: true,
```

Comment:
`injectHot` injects the code for enabling the hot load service. `injectClient` injects the client that actually starts the connection. Reference: https://github.com/gaearon/react-hot-loader/issues/141